packages/ui/src/DateTimeField.tsx

```diff
@@ -25,7 +24,7 @@ export const DateTimeField = ({
         {mode === "datetime" && (
           <DateTimePickerWeb disableClock value={value} onChange={onChange} />
         )}
-        {mode === "date" && <DatePicker value={value} onChange={onChange} />}
+        {/* {mode === "date" && <DatePicker value={value} onChange={onChange} />} */}
```

Comment:
Is there a reason to comment this out as part of this PR?

Reply:
Ah that was causing issues, but I think that was from basing it on the mono repo PR (which does not like the date picker for whatever reason). I'll remove!